charts/kyverno/templates/deployment.yaml

```diff
@@ -32,17 +32,19 @@ spec:
       {{- end }}
       {{- if or .Values.antiAffinity.enable .Values.podAffinity .Values.nodeAffinity }}
       affinity:
-        {{- if and .Values.antiAffinity.enable .Values.podAntiAffinity }}
+        {{- if and .Values.antiAffinity.enable }}
```

Comment:
Should it just be `if ...` without the `and` since only one value being evaluated?

Reply:
Good catch !

Reply:
Fixed